### PR TITLE
Simple refactor 

### PR DESCRIPTION
**Pull request by shefass**

### CODE REVIEW

File: src/main/java/btdex/ui/Main.java [hunk 1]
```diff
@@ -511,6 +505,10 @@ public void update() {
 	}
 
 	private void updateUI() {
+		Icon ICON_CONNECTED = i.get(Icons.CONNECTED);
```
Comment (jjos2372) [inline src/main/java/btdex/ui/Main.java:508]: This would update icons inside the update thread (running continuously). I don't think this is a good idea.

**Comment by shefass** [comment]
ok, I will look into it

2020-05-14, kt, 02:59 jjos2372 <notifications@github.com> rašė:

**Comment by shefass** [comment]
I wrote more code to Main class, so I will close this PR and will create new PR with your feedback

File: src/main/java/btdex/ui/Main.java (hunk 1 above)
Comment (shefass) [reply src/main/java/btdex/ui/Main.java:508]: I agree, no need to update it

**Pull request closed without merging**